internal/ingress/controller/nginx.go

```diff
@@ -106,7 +107,7 @@ func NewNGINXController(config *Configuration, fs file.Filesystem) *NGINXControl
 		}),
 
 		stopCh:   make(chan struct{}),
-		updateCh: make(chan store.Event, 1024),
+		updateCh: channels.NewRingChannel(4096),
```

Comment:
does this specify a limit of 4096? If so, the initial load does not appear to pop items off the ring until the cache has been initialized. This still might cause a problem. Can we lower to 1024 to confirm?

Reply:
yes, let me lower that value and create a different docker image

Reply:
cool. still works